### ISSUE TITLE
Cura does not start slicing, and there's no print timer displayed.

### ISSUE THREAD

**Issue by laban93** [Ultimaker/Cura#937]
The cura program does freeze up quite often, so i need to exit and relaunch the program. Doing that several times before it recognises the model and starts slicing. The print timer is also set to 00h 00m.

**Comment by Ghostkeeper** [comment]
Stuck in "Preparing to slice..."?

**Comment by robygladio** [comment]
Same problem there.
before I can slice without problem now is an infinite preparing for slicing..
curaengine is set as RUN AS ADMINISTRATOR, no firewall

**Comment by Ghostkeeper** [comment]
That's a duplicate of #931.

**Comment by 3DKernow** [comment]
I'm having the same problem with 2.1.3. Some stls slice OK. Others don't.

**Comment by 3DKernow** [comment]
I've found the answer for my problem from a comment in another discussion strand, If you have set the software to support printing one at a time (which I had done), then the software has problems with designs above a certain height, because (I assume) the software needs to stop the nozzle bumping into a finished object. I changed the setting and my problem stl immediately sliced.

**Comment by 3DKernow** [comment]
.... and I had another stl which was giving slicing problems. This was a design I had pulled off Thingiverse and it had downloaded as a very small object (11*17*5 mm), Once I scaled it up, it started slicing fine. Just doubling the size did the trick.
CONCLUSIONS:  slicing is sensitive when files are scaled down too much. There is also at least one setting (print one at a time) which affects files which are taller than a certain height.
